ibm_catalog.json

```diff
@@ -0,0 +1,543 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-vpc",
+      "label": "[Add-ons Beta] Cloud automation for VPC",
+      "product_kind": "solution",
+      "tags": [
+        "network_vpc",
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "support_ibm",
```

Comment:
@maheshwarishikha I think we need to remove this and add the `ibm_beta` tag. We are not supporting this tile via IBM Cloud - its only experimental beta